src/runtime/builtin_modules/sys.cpp

```diff
@@ -816,6 +816,8 @@ void setupSys() {
     sys_flags_cls->giveAttr(
         "__new__", new BoxedFunction(FunctionMetadata::create((void*)BoxedSysFlags::__new__, UNKNOWN, 1, true, true)));
     sys_flags_cls->tp_dealloc = (destructor)BoxedSysFlags::dealloc;
+
+    sys_flags_cls->giveAttr("ignore_environment", boxInt(Py_IgnoreEnvironmentFlag));
```

Comment:
I don't like this half implemented `sys.flags` thing but I don't think you change makes it worse so I'm fine with merging this but I created https://github.com/dropbox/pyston/issues/1304 to make sure we fix it at some point